### PR TITLE
11.0 merge from 10.0

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: project_timelog/__manifest__.py [hunk 1]
```diff
@@ -36,5 +35,5 @@
     "post_init_hook": None,
 
     "auto_install": False,
-    "installable": False,
+    "installable": True,
```
Comment (yelizariev) [inline project_timelog/__manifest__.py]: Keep it disabled too

**Review by yelizariev — CHANGES_REQUESTED** [review]
If tests will work without project_timelog, we can postpone the issue https://github.com/it-projects-llc/misc-addons/issues/818

File: project_timelog/__manifest__.py (hunk 1 above)
Comment (em230418) [reply project_timelog/__manifest__.py]: Done https://github.com/it-projects-llc/misc-addons/pull/813/commits/75ebb545dd73c548b76d97037fcfd75ac19d260c

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**